matter/wifi/brd4186c.h

```diff
@@ -23,31 +24,33 @@
 
 #define MY_USART EUSART1
 #define MY_USART_CLOCK cmuClock_EUSART1
-#define MY_USART_TX_SIGNAL dmadrvPeripheralSignal_EUSART1_TXBL //dmadrvPeripheralSignal_USART0_TXBL
-#define MY_USART_RX_SIGNAL dmadrvPeripheralSignal_EUSART1_RXDATAV //dmadrvPeripheralSignal_USART0_RXDATAV
-
 
-#define SPI_CLOCK_PIN PIN(C, 3)
-#define SPI_MOSI_PIN  PIN(C, 1)
-#define SPI_MISO_PIN  PIN(C, 2)
-#define SPI_CS_PIN    PIN(C, 0)
+#define MY_USART_TX_SIGNAL                                                     \
+	  dmadrvPeripheralSignal_EUSART1_TXBL // dmadrvPeripheralSignal_USART0_TXBL
+#define MY_USART_RX_SIGNAL                                                     \
+	  dmadrvPeripheralSignal_EUSART1_RXDATAV // dmadrvPeripheralSignal_USART0_RXDATAV
```

Comment:
Are the comments necessary here? seem like something that can be removed

Reply:
done